Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -787,6 +786,37 @@ private Task OnColumnDragEnded( DragEventArgs e )
         return Task.CompletedTask;
     }
 
+    private Task OnColumnDropped( DataGridColumn<TItem> columnBeingDropped )

```

Comment:
`columnBeingDropped`
Is it the one that is dragged and now it's dropping? or is it the one that is registering the drop event on itself?

I would call them `columnTarget` and `collumnDragging`. or somethin similar.